### ISSUE TITLE
push notifications stopped working

### ISSUE THREAD

**Issue by matteocnt92** [go-vikunja/app#98]
Hi, when I add a task with a due date and a reminder, the app does not show any notification when the task is due.
Just a couple of days ago it was working fine, on the same vikunja instance v. 0.24.4
I already tried reinstalling the app (v. 0.1.8). The test notification is working fine

Samsung Galaxy s24 ultra, android 14, october security patch

**Comment by Benimautner** [comment]
If it worked a couple days ago, what changed?

And make sure to try to schedule a task notification a couple minutes in the future to give the app time to schedule it.

**Comment by matteocnt92** [comment]
I'm still testing, since I correctly received a notification for another task. I will let you know as soon as I find something useful

**Comment by Marck** [comment]
Any updates?

**Comment by matteocnt92** [comment]
notifications still work randomly, sometimes they do, sometimes they don't, with exactly the same settings inside tasks.
This is even after a factory reset of the phone after installing Android 15 and oneui 7. The app has already been excluded from all android/samsung battery optimizations
I always receive email notifications though

**Comment by kolaente** [comment]
Is this still a problem with the latest unstable build?

**Comment by matteocnt92** [comment]
Yes. The app version used when opening the bug was 0.1.8, and it’s still the same version in use

**Comment by kolaente** [comment]
Which version did you download? The unstable builds still show 0.1.8 even though they're actually newer.

**Comment by matteocnt92** [comment]
I downloaded version 0.1.8 from the releases page. Where can I find the updated unstable build?

**Comment by kolaente** [comment]
Either the [Google Play Beta](https://play.google.com/store/apps/details?id=io.vikunja.app) or any of the [latest CI build artefacts](https://github.com/go-vikunja/app/actions/workflows/ci.yml?query=branch%3Amain) (at the bottom of the link).
